diff --git a/lua/plugins/core.lua b/lua/plugins/core.lua
index ca122d0..53cd25f 100644
--- a/lua/plugins/core.lua
+++ b/lua/plugins/core.lua
@@ -109,8 +109,8 @@ return {
   {
     "numToStr/Comment.nvim",
     keys = {
-      { "gc", mode = { "n", "v" }, desc = "Comment toggle linewise" },
-      { "gb", mode = { "n", "v" }, desc = "Comment toggle blockwise" },
+       { "gc", mode = { "n", "v" }, desc = "Comment toggle linewise" },
+       -- { "gb", mode = { "n", "v" }, desc = "Comment toggle blockwise" },
     },
     opts = function()
       local commentstring_avail, commentstring = pcall(require, "ts_context_commentstring.integrations.comment_nvim")
diff --git a/lua/plugins/neo-tree.lua b/lua/plugins/neo-tree.lua
index d4da5fa..6447ad7 100644
--- a/lua/plugins/neo-tree.lua
+++ b/lua/plugins/neo-tree.lua
@@ -101,7 +101,7 @@ return {
       end,
     },
     window = {
-      width = 30,
+      width = 40,
       mappings = {
         ["<space>"] = false, -- disable space until we figure out which-key disabling
         ["[b"] = "prev_source",
